src/as_execution/common.rs

```diff
@@ -38,6 +38,8 @@ pub(crate) fn call_module(
             )))
         })?;
 
+    interface.increment_recursion_counter()?;
```

Comment:
doesn't that double-count  with the increment that happens also in `init_call` ?

Reply:
removed the one in init_call